### PR TITLE
clean up grid classes

### PR DESCRIPTION
**Pull request by spadgett**
/assign @rhamilto

### CODE REVIEW

File: frontend/public/components/machine-set.tsx [hunk 1]
```diff
@@ -69,9 +69,9 @@ export const getReadyReplicas = (machineSet: MachineSetKind | MachineDeploymentK
 export const getAvailableReplicas = (machineSet: MachineSetKind | MachineDeploymentKind) => _.get(machineSet, 'status.availableReplicas', 0);
 
 const tableColumnClasses = [
-  classNames('pf-m-4-col-on-md', 'pf-m-6-col-on-sm'),
-  classNames('pf-m-4-col-on-md', 'pf-m-6-col-on-sm'),
-  classNames('pf-m-4-col-on-md', 'pf-m-hidden', 'pf-m-visible-on-md'),
+  classNames('col-sm-4', 'col-xs-6'),
```
Comment (benjaminapetersen) [inline frontend/public/components/machine-set.tsx]: I'm inclined to wonder if there are several exact matches in these `tableColumnClasses` sets that could be shared. For example, this looks the same as the one in `machine-deployment`, `limit-range`, `default-resource`, `cluster-service-plan`...

Comment (benjaminapetersen) [reply frontend/public/components/machine-set.tsx]: This is obviously a good step, so I wouldn't hold the PR on it.

**Comment by priley86** [comment]
since you are still currently using BS3 grid elsewhere, and we are not considering a 12 column flexbox grid in PF4, I think this is fine.

However, you can still consider a PF4 CSS Grid in the future. It should work technically the same way (and it seems to, for me). Here is one such example:
https://github.com/priley86/web-ui/commit/5229ed0eb86ab9d3bf8b2e139d7f25f46b5eb81f

Maybe you can try this globally in the future...

**Comment by rhamilto** [comment]
> Maybe you can try this globally in the future...

IMO, until we decide to remove Boostrap's grid entirely, we should avoid using PF4's grid.  Let's use and maintain one solution, not two.

**Pull request merged**